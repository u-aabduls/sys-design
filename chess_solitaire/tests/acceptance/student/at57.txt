-- ACCEPTANCE TEST #57: SUCCESS CASE: KNIGHT NOT BLOCKED - #2

-- Setup some chess pieces for the game
setup_chess (N,3,3)
setup_chess (Q,2,2)
setup_chess (B,1,2)

start_game

move_and_capture(3,3,1,2)