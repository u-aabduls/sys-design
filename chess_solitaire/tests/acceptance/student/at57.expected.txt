  # of chess pieces on board: 0
  Game being Setup...
  ....
  ....
  ....
  ....
->setup_chess(N,3,3)
  # of chess pieces on board: 1
  Game being Setup...
  ....
  ....
  ..N.
  ....
->setup_chess(Q,2,2)
  # of chess pieces on board: 2
  Game being Setup...
  ....
  .Q..
  ..N.
  ....
->setup_chess(B,1,2)
  # of chess pieces on board: 3
  Game being Setup...
  .B..
  .Q..
  ..N.
  ....
->start_game
  # of chess pieces on board: 3
  Game In Progress...
  .B..
  .Q..
  ..N.
  ....
->move_and_capture(3,3,1,2)
  # of chess pieces on board: 2
  Game In Progress...
  .N..
  .Q..
  ....
  ....
